emgena/serializers.py

```diff
@@ -58,12 +58,26 @@ class Meta:
 
 
 class NotifySerializer(serializers.Serializer):
-
-    from_email = serializers.EmailField(max_length=200, required=True)
+    from_email = serializers.CharField(max_length=200, required=True)
+    cc = serializers.CharField(max_length=200, allow_blank=True, required=False)
     subject = serializers.CharField(max_length=500, required=True)
     message = serializers.CharField(max_length=1000, required=True)
     is_consent = serializers.BooleanField(default=False, required=False)
 
+    def is_valid(self):
+        for field in ['from_email', 'cc']:
+            if field not in self.initial_data:
+                continue
+            for email in self.initial_data[field].split(','):
```

Comment:
Is it worth doing a `email = email.strip()` here just in case there is leading/trailing whitespace?

Reply:
Sure it is. I know I think about it but I guess I forgot. I'll commit a change with it